PV260_Project/DataLayer/Models/Email.cs

```diff
@@ -9,5 +9,12 @@ namespace DataLayer.Models
     public class Email : BaseEntity
     {
         public string Address { get; set; }
+        
+        public override string ToString()
+        {
+            return $"Id: {Id}, address: {Address}";
```

Comment:
As we already talked about this before and also Radim agreed, the domain models (those that are in DB layer) should not contain any logic. But I guess its no big deal when we are finishing the project :D

Reply:
I actually removed the `ToString()` method from here, and added `Messages.PrintEmail()`, since otherwise the same would have to be done for DataSet when testing it (so that we avoid adding more 'incorrect' code while being consistent).